src/mlagility/analysis/analysis.py

```diff
@@ -80,7 +80,6 @@ def call_benchit(
     """
     Calls the benchit function from within the model forward function
     """
-
```

Comment:
nit: any reason for removing this whitespace?

Reply:
Adding it back.